.github/workflows/image.yaml

```diff
@@ -0,0 +1,49 @@
+# (C) Copyright Red Hat 2022.
```

Comment:
nit: Is the above copyright coming from the github action ?
If not then better to change to "Confidential Containers Contributors"

Reply:
I was also confused initially so copied the same from other github actions yaml in the repo. Let me update.